assets/css/components/_announcements.scss

```diff
@@ -22,16 +22,28 @@
   }
 }
 
-.announcement-metadata {
+.announcement-list-item-metadata {
   color: $secondary-font-color;
   font-size: $small-font-size;
+}
+
+.announcement-metadata-container {
+  color: $secondary-font-color;
+  display: flex;
+  flex-wrap: wrap;
+  font-size: $small-font-size;
+
+  > * {
```

Comment:
It’d be great to avoid wildcard selectors like this, in favor of applying CSS classes directly to the items we want to style. It helps to reduce specificity and is a bit more clear, I think.